index.bs

```diff
@@ -3246,22 +3235,30 @@ Issue: Specify this in terms of <a href="https://html.spec.whatwg.org/multipage/
 
 # Report delivery # {#report-delivery}
 
-The user agent MUST periodically [=set/iterate=] over its [=event-level report cache=] and
-[=aggregatable report cache=] and run [=queue a report for delivery=] on each item.
+The user agent MUST periodically run [=queue reports for delivery=] on the
+[=event-level report cache=] and [=aggregatable report cache=].
+
+To <dfn>queue reports for delivery</dfn> given a [=set=] of
+[=attribution reports=] |cache|, run the following steps [=in parallel=]:
```

Comment:
Hm, but we probably don't want to touch the report cache in parallel since that can lead to data races, right? I think step one here should be executed on the same "event sequence" as every other algorithm that touches the report caches.

Reply:
Yes, in general, the entire spec needs to be audited to ensure correctness around exclusive access to all of the attribution caches. I'm not sure what the best way to do that is, but I don't know if that should block this PR, since the existing logic already suffers from the same issue.

It seems that @linnan-github made a [related change](https://github.com/WICG/attribution-reporting-api/pull/1108) recently.

I'm of the opinion that we may want to have a single multi-producer/single-consumer parallel queue containing source/trigger registrations and these periodic report-delivery attempts.

Thoughts?

Reply:
Agree we should audit, but I guess my point was that I think just omitting "in parallel" from this step is arguably more correct than the current PR.